langchain/evaluation/__init__.py

```diff
@@ -1,29 +1,41 @@
-"""Functionality relating to evaluation.
+"""Evaluation chains for grading LLM and Chain outputs.
 
-This module contains off-the-shelf evaluation chains for
-grading the output of LangChain primitives such as LLMs and Chains.
+This module contains off-the-shelf evaluation chains for grading the output of
+LangChain primitives such as language models and chains.
+
+To load an evaluator, you can use the :func:`load_evaluators <langchain.evaluation.loading.load_evaluators>` function with the
+name of the evaluator to load.
+
+To load one of the LangChain HuggingFace datasets, you can use the :func:`load_dataset <langchain.evaluation.loading.load_dataset>` function with the
+name of the dataset to load.
 
 Some common use cases for evaluation include:
 
-- Grading accuracy of a response against ground truth answers: QAEvalChain
-- Comparing the output of two models: PairwiseStringEvalChain
-- Judging the efficacy of an agent's tool usage: TrajectoryEvalChain
-- Checking whether an output complies with a set of criteria: CriteriaEvalChain
+- Grading the accuracy of a response against ground truth answers: :class:`QAEvalChain <langchain.evaluation.qa.eval_chain.QAEvalChain>`
+- Comparing the output of two models: :class:`PairwiseStringEvalChain <langchain.evaluation.comparison.eval_chain.PairwiseStringEvalChain>`
+- Judging the efficacy of an agent's tool usage: :class:`TrajectoryEvalChain <langchain.evaluation.agents.trajectory_eval_chain.TrajectoryEvalChain>`
+- Checking whether an output complies with a set of criteria: :class:`CriteriaEvalChain <langchain.evaluation.criteria.eval_chain.CriteriaEvalChain>`
+
+This module also contains low-level APIs for creating custom evaluators for
+specific evaluation tasks. These include:
 
-This module also contains low level APIs for making more evaluators for your
-custom evaluation task. These include:
-- StringEvaluator: Evaluates an output string against a reference and/or
-    with input context.
-- PairwiseStringEvaluator: Evaluates two strings against each other.
-"""
+- :class:`StringEvaluator <langchain.evaluation.schema.StringEvaluator>`: Evaluates an output string against a reference and/or input context.
+- :class:`PairwiseStringEvaluator <langchain.evaluation.schema.PairwiseStringEvaluator>`: Evaluates two strings against each other.
 
+"""  # noqa: E501
 from langchain.evaluation.agents.trajectory_eval_chain import TrajectoryEvalChain
-from langchain.evaluation.comparison import PairwiseStringEvalChain
+from langchain.evaluation.comparison.eval_chain import PairwiseStringEvalChain
```

Comment:
nit: import suffixed with `.eval_chain` looks a bit funny, imo better to move it up one level like you had before